scripts/remove

```diff
@@ -1,40 +1,49 @@
 #!/bin/bash
+
 #=================================================
-# GENERIC STARTING
+# GENERIC START
 #=================================================
 # IMPORT GENERIC HELPERS
 #=================================================
+
+source _common.sh
 source _future.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
 # LOAD SETTINGS
 #=================================================
+
 readonly app=$YNH_APP_INSTANCE_NAME
 readonly domain=$(ynh_app_setting_get "$app" domain)
 readonly final_path=$(ynh_app_setting_get "$app" final_path)
 
 #=================================================
 # STANDARD REMOVE
 #=================================================
-# STOP AND REMOVE SERVICE
+# REMOVE SERVICE FROM ADMIN PANEL
 #=================================================
 
-ynh_remove_systemd_config
+# Remove a service from the admin panel, added by `yunohost service add`
+if yunohost service status | grep -q $app
+then
+	echo "Remove $app service"
+	yunohost service remove $app
+fi
+
+yunohost service remove mongodb
```

Comment:
Unfortunately I don't think we can simply remove mongodb here ... We need to make sure that no other app uses mongodb, and I'm not sure we have a solution for this at the moment :/

Idk ... we can keep this removal if we check that not so many popular apps use mongodb but we need to be careful with this othewise we might break some apps.